instaloader/structures.py

```diff
@@ -128,6 +128,13 @@ def mediaid(self) -> int:
         """The mediaid is a decimal representation of the media shortcode."""
         return int(self._node['id'])
 
+    @property
+    def title(self) -> Optional[str]:
+        """Title of post"""
+        if "title" in self._full_metadata_dict:
+            return self._full_metadata_dict["title"]
+        return None
```

Comment:
```suggestion
        try:
            return self._field('title')
        except KeyError:
            return None
```

Otherwise, the access to `self._full_metadata_dict` triggers a full metadata fetch, even if the data is already contained in `self._node`.